templates/download/xilinx/index.html

```diff
@@ -14,13 +16,11 @@
       <h1>Install Ubuntu on Xilinx</h1>
       <p>Run Ubuntu on your Xilinx Zynq UltraScale+ MPSoC-based evaluation boards and Kria production SOMs. </p>
       <p>Pick the OS  image to match your hardware, flash it onto SD/microSD card, load it onto your board and away you go.</p>
-      <p>If you have any questions about the supported platforms or would like information about our certification program, please contact us.</p>
-      <a href="/download/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
     </div>
     <div class="col-6 u-align--center u-hide--small">
       {{ image (
         url="https://assets.ubuntu.com/v1/8a8c250f-xilinx-logo.svg",
-        alt="",
+        alt="Xilinx Logo",
```

Comment:
This `alt` text is not useful for users with screenreaders. I think empty is best here so its not read out.

Reply:
I agree, but it was specified by @davegoddard42 so I wonder if there was a reason?

Reply:
I might have been a bit over zealous with the alt text! I've removed it from the doc as per Ant's suggestion.